src/modules/Dropdown/Dropdown.js

```diff
@@ -39,7 +124,7 @@ export default class Dropdown extends Component {
 
   render() {
     const options = _.map(this.props.options, (opt, i) => {
-      return <option key={i} value={opt.value}>{opt.text}</option>;
+      return <option key={opt.value} value={opt.value}>{opt.text}</option>;
```

Comment:
I swear React used to throw warnings in the console for this and then stopped. Not sure why they stopped.